doc/source/whatsnew/v0.22.0.txt

```diff
@@ -69,8 +69,9 @@ Removal of prior version deprecations/changes
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Indexers on ``Series`` or ``DataFrame`` no longer create a reference cycle (:issue:`17956`)
+- Indexers on Series or DataFrame no longer create a reference cycle (:issue:`17956`)
 - Added a keyword argument, ``cache``, to :func:`to_datetime` that improved the performance of converting duplicate datetime arguments (:issue:`11665`)
+- DatetimeIndex.date skips the iterations that create many python objects (:issue:`18058`)
```

Comment:
``Improved performance of :func:`Series.dt.date` and :func:`DatetimeIndex.date` ``